packages/diracx-web-components/components/Login/LoginForm.tsx

```diff
@@ -129,109 +126,105 @@ export function LoginForm({
           sx={{
-            ml: { xs: "5%", md: "30%" },
-            mr: { xs: "5%", md: "30%" },
+            width: "100%",
+            display: "flex",
+            justifyContent: "center",
+            paddingTop: "10%",
+            paddingBottom: "10%",
           }}
         >
-          <Box
+          <img src={logoURL} alt="DIRAC logo" width={150} height={150} />
+        </Box>
+        {singleVO ? (
+          <Typography
+            variant="h3"
+            gutterBottom
+            sx={{ textAlign: "center" }}
+            data-testid="h3-vo-name"
+          >
+            {selectedVO}
+          </Typography>
+        ) : (
+          <Autocomplete
+            options={Object.keys(metadata.virtual_organizations)}
+            getOptionLabel={(option) => option}
+            renderInput={(params) => (
+              <TextField
+                {...params}
+                label="Select Virtual Organization"
+                variant="outlined"
+                data-testid="autocomplete-vo-select"
+              />
+            )}
+            value={selectedVO}
+            onChange={handleVOChange}
             sx={{
-              width: "100%",
-              display: "flex",
-              justifyContent: "center",
-              paddingTop: "10%",
-              paddingBottom: "10%",
+              "& .MuiAutocomplete-root": {
+                // Style changes when an option is selected
+                opacity: selectedVO ? 0.5 : 1,
+              },
             }}
-          >
-            <img src={logoURL} alt="DIRAC logo" width={150} height={150} />
-          </Box>
-          {singleVO ? (
+          />
+        )}
+        {selectedVO && (
+          <Box sx={{ mt: 4 }}>
+            <FormControl fullWidth>
+              <InputLabel>Select a Group</InputLabel>
+              <Select
+                name={selectedVO}
+                value={
+                  selectedGroup ||
+                  metadata.virtual_organizations[selectedVO].default_group
+                }
+                label="Select a Group"
+                onChange={handleGroupChange}
+                data-testid="select-group"
+              >
+                {Object.keys(
+                  metadata.virtual_organizations[selectedVO].groups,
+                ).map((groupKey) => (
+                  <MenuItem key={groupKey} value={groupKey}>
+                    {groupKey}
+                  </MenuItem>
+                ))}
+              </Select>
+            </FormControl>
+            <Stack
+              direction={{ xs: "column", sm: "row" }}
+              spacing={2}
+              sx={{ mt: 5, width: "100%" }}
+            >
+              <Button
+                variant="contained"
+                sx={{
+                  flexGrow: 1,
+                }}
+                onClick={handleConfigurationChanges}
+                data-testid="button-login"
+              >
+                Login through your Identity Provider
```

Comment:
```suggestion
                Login via your Identity Provider
```

this is just a suggestion

Reply:
Thanks, I will apply it :slightly_smiling_face: